scripts/build/gen_isr_tables.py

```diff
@@ -27,6 +28,13 @@
 
 INTERRUPT_BITS = [8, 8, 8]
 
+swt_spurious_handler = "((uintptr_t)&z_irq_spurious)"
+swt_shared_handler = "((uintptr_t)&z_shared_isr)"
```

Comment:
We could, but the problem is that internally we are using uintptr_t and are always casting to uintptr_t. I had saw it in a few places where the interrupt handler without an argument was used. It works in such approach and would throw an error during compilation otherwise. I also think we should use the proper type here, but I did now want to change others implementations - For this PR it was important for me to be able to just compare the binary files with new and current approach. And not changing it right now gives such possibility.

Reply:
@rakons, on your last point about comparing before/after, what are you seeing with this approach?